### PR TITLE
Bump version from 0.6.3 to 0.7.0.dev0 in docs and .yaml

### PR DESCRIPTION
**Pull request by pcmoritz**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@
 .. image:: https://readthedocs.org/projects/ray/badge/?version=latest
     :target: http://ray.readthedocs.io/en/latest/?badge=latest
 
-.. image:: https://img.shields.io/badge/pypi-0.6.3-blue.svg
+.. image:: https://img.shields.io/badge/pypi-0.7.0dev0-blue.svg
```
Comment (robertnishihara) [inline README.rst]:
```suggestion
.. image:: https://img.shields.io/badge/pypi-0.6.3-blue.svg
```

I think we shouldn't update this one.

**Comment by robertnishihara** [comment]
@pcmoritz the URL that you use in this PR does not work. I'm a bit confused.

There are three different names.

1. **0.7.0dev0** (this is the name you use in this PR).
2. **0.7.0dev** (this is the version string in `__init__.py`).
3. **0.7.0.dev0** (this is the name of the wheels in the S3 bucket.).

Any idea what's going on here?

**Comment by pcmoritz** [comment]
Interesting, I'm seeing this warning during in the linux wheel travis entry:
```
/opt/python/cp27-cp27mu/lib/python2.7/site-packages/setuptools/dist.py:388: UserWarning: Normalizing '0.7.0dev' to '0.7.0.dev0'
```

So it looks like we need to use 0.7.0.dev0. I'll change it.

**Comment by pcmoritz** [comment]
Ok I updated it, good catch. Now downloading the wheels seems to work!

**Review by robertnishihara — APPROVED** [review]

**Pull request merged**